api/metrics_service.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "google/api/annotations.proto";
+import "metrics.proto";
+
+// Service for streaming metrics to connected end point.
```

Comment:
Can you please change the wording here to describe in more detail what the endpoint is, and also update `README.md` in the root directory to describe this new service that we are connecting to?

Reply:
Will change..